examples/reduction/examples/histogram.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2023 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+
+import cunumeric as np
+from reduction import (
+    bincount,
+    categorize,
+    histogram,
+    print_store,
+    to_cunumeric_array,
+    user_context,
+)
+
+import legate.core.types as ty
+
+
+def test(size: int, num_bins: int):
+    input = user_context.create_store(ty.int32, size)
+    to_cunumeric_array(input)[:] = np.random.randint(
```

Comment:
Same comment here. It was a bit weird to me to create a cunumeric array from the store, then assign to a slice of it from a numpy array. Might be more intuitive to create a cunumeric random array and then have a `get_cunumeric_store` helper function?